src/runtime/schema/settings.ts

```diff
@@ -33,42 +43,85 @@ export type SettingsInput = {
     /**
      * todo
      */
-    default?: ConnectionConfig | false
+    default?: ConnectionSettings | false
     // Extra undefined below is forced by it being above, forced via `?:`.
     // This is a TS limitation, cannot express void vs missing semantics,
     // being tracked here: https://github.com/microsoft/TypeScript/issues/13195
-    [typeName: string]: ConnectionConfig | undefined | false
+    [connectionTypeName: string]: ConnectionSettings | undefined | false
   }
   /**
    * Should a [GraphQL SDL file](https://www.prisma.io/blog/graphql-sdl-schema-definition-language-6755bcb9ce51) be generated when the app is built and to where?
    *
    * A relative path is interpreted as being relative to the project directory.
+   *
    * Intermediary folders are created automatically if they do not exist
    * already.
    *
-   * @default false
+   * @default "api.graphql"
    */
   generateGraphQLSDLFile?: false | string
   /**
-   * A glob pattern which will be used to find the files from which to extract the backing types used in the `rootTyping` option of `schema.(objectType|interfaceType|unionType|enumType)`
+   * A glob pattern which will be used to find the files from which to extract the backing types. Backing types are used in/selected in  the rootTyping option of these schema methods:
+   *
+   * ```
+   * schema.objectType
+   * schema.interfaceType
+   * schema.unionType
+   * schema.enumType
+   * ```
+   *
+   * Here is a mini glob syntax guide (copied from [node-glob](https://github.com/isaacs/node-glob#glob-primer)). The following characters have special meaning in path portions:
+   *
+   * ```
+   * * –– Matches 0 or more characters in a single path portion
+   *
+   * ? –– Matches 1 character
+   *
+   * [...] –– Matches a range of characters, similar to a RegExp range. If the first character of the range is ! or ^ then it matches any character not in the range.
+   *
+   * !(pattern|pattern|pattern) –– Matches anything that does not match any of the patterns provided.
+   *
+   * ?(pattern|pattern|pattern) –– Matches zero or one occurrence of the patterns provided.
+   *
+   * +(pattern|pattern|pattern) –– Matches one or more occurrences of the patterns provided.
+   *
+   * *(a|b|c) –– Matches zero or more occurrences of the patterns provided
+   *
+   * AT_SYMBOL(pattern|pat*|pat?erN) –– Matches exactly one of the patterns provided
```

Comment:
https://stackoverflow.com/questions/61893953/how-to-escape-symbol-in-jsdoc-for-vscode